example/ios/Podfile

```diff
@@ -1,25 +1,8 @@
-$deployment_target = '9.0'
-platform :ios, $deployment_target
+require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
-def add_flipper_pods!
-  version = '~> 0.33.1'
-  pod 'FlipperKit', version, :configuration => 'Debug'
-  pod 'FlipperKit/FlipperKitLayoutPlugin', version, :configuration => 'Debug'
-  pod 'FlipperKit/SKIOSNetworkPlugin', version, :configuration => 'Debug'
-  pod 'FlipperKit/FlipperKitUserDefaultsPlugin', version, :configuration => 'Debug'
-  pod 'FlipperKit/FlipperKitReactPlugin', version, :configuration => 'Debug'
-end
-# Post Install processing for Flipper
-def flipper_post_install(installer)
-  installer.pods_project.targets.each do |target|
-    if target.name == 'YogaKit'
-      target.build_configurations.each do |config|
-        config.build_settings['SWIFT_VERSION'] = '4.1'
-      end
-    end
-  end
-end
+$deployment_target = '10.0'
```

Comment:
This is the new default. We can roll back but Flipper requires 10, so we would have to use an older version. What do you think?

Reply:
well, this is just the example, so it's fine to move the deployment target. 
and in any case, there really isn't a lot of reason left to support iOS 9.0 at this point, the user base is getting smaller and smaller.